osu.Game.Rulesets.Tau/UI/TauHardButtonTouchInputMapper.cs

```diff
@@ -0,0 +1,206 @@
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osuTK;
+
+namespace osu.Game.Rulesets.Tau.UI;
+
+// basically copies CatchTouchInputMapper, without the right side buttons
+internal partial class TauHardButtonTouchInputMapper(TauInputManager inputManager) : VisibilityContainer
```

Comment:
```suggestion
public partial class TauHardButtonTouchInputMapper : VisibilityContainer
```

For consistency, the constructor of the class should be placed inside somewhere, like so:
```cs
public TauHardButtonTouchInputMapper(TauInputManager inputManager) {
    this.inputManager = inputManager;
}
```